jenkins/ci.suse.de/templates/cloud-socok8s-integration.yaml

```diff
@@ -101,11 +58,6 @@
           default: 'airship'
           description: >-
             Which deployment mechanism?
-      - bool:
-          name: run_tempest
-          default: '{run_tempest|true}'
-          description: >-
-            Run tempest tests?
       - choice:
           name: tempest_test_type
           choices:
```

Comment:
Shouldn't default be "smoke" here, until we're good to move to "all"?

Reply:
This are the daily jobs, nothing to do with our PR ones :D

Reply:
Yeah, that's why I was thinking of a phased approach: If our PRs don't test this, it could be inherently broken, with nobody having a look. So I doubt the interest of having those immediately. I thought once we could be in a better shape with good trends, then we could upgrade the daily testing to increase coverage, and slowly include the results into PR testing. But that's good enough as is.